### PR TITLE
feat(component): add class card component

### PR DESCRIPTION
**Pull request by herman-rogers**
#42
Screen Shot 2022-05-16 at 7 18 39 PM

### CODE REVIEW

**Review by awpala — DISMISSED** [review]
Looks great as a general starting point, nice work!

File: src/stories/ClassCard.stories.tsx [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+					reviewValue={4}
+					difficulty={2.0}
+					image='/static/omshub-stub.jpg'
+				></ClassCard>
+			</Grid>
+		</Grid>
+	</>
+)
+
+export const Primary = Template.bind({})
```
Comment (ctran4347) [inline src/stories/ClassCard.stories.tsx:43]: The component looks good! Can you possibly add more [controls](https://storybook.js.org/docs/react/essentials/controls) to the component?

Comment (herman-rogers) [reply src/stories/ClassCard.stories.tsx:43]: Updated the component to have all controls

**Review by awpala — APPROVED** [review]

**Review by knakamura13 — APPROVED** [review]
This PR is fantastic. Thanks for your hard work in cleaning up the template code as well as adding new content.

**Pull request merged**